test-cases/scylla-operator/longevity-scylla-operator-many-clients-4h.yaml

```diff
@@ -0,0 +1,33 @@
+test_duration: 360
+
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=20 -pop seq=1..20971520 -col 'n=FIXED(2) size=FIXED(256)' -log interval=5"
+
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=240m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=4 -pop seq=1..20971520 -col 'n=FIXED(2) size=FIXED(256)' -log interval=5",
+             "cassandra-stress read  cl=QUORUM duration=240m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=4 -pop seq=1..20971520 -col 'n=FIXED(2) size=FIXED(256)' -log interval=5"
+             ]
+
+n_db_nodes: 5
+k8s_n_scylla_pods_per_cluster: 4
+n_loaders: 50
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.metal'
+instance_type_loader: 'c5.large'
+instance_type_monitor: 'm5.2xlarge'
+instance_type_runner: 'r5.large'
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '031'
+nemesis_multiply_factor: 1
+nemesis_interval: 5
+nemesis_filter_seeds: false
+seeds_num: 3
+round_robin: true
+keyspace_num: 50
+use_mgmt: false
+k8s_deploy_monitoring: true
```

Comment:
Better to disable the `k8s_deploy_monitoring` option.
It is useless.

Reply:
why is it useless? do we have some issue about it?

Reply:
It deploys second separate monitoring on K8S. No one uses it. Takes time and resources.
Useless is useless.

Reply:
It's soon gonna be remove/replaced.

But we didn't get to it.

Reply:
removed